analytics-webapps/src/main/webapp/vue-app/breadcrumb-portlet/components/AnalyticsDrawerNavigation.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <exo-drawer
+    id="drawerPageAnalytics"
+    ref="analyticsPage"
+    right>
+    <template slot="title">
+      {{ $t('analytics.drawer') }}
+    </template>
+    <template slot="content">
+      <v-treeview
+        class="treeViewPage"
+        activatable
+        dense
+        hoverable
+        :items="listPage">
+        <template slot="label" slot-scope="{ item }">
+          <a
+            class="pageNameAnalytics"
+            :href="getPageUri(item)">{{ item.label }}</a>
+        </template>
+      </v-treeview>
+    </template>
+  </exo-drawer>
+</template>
+<script>
+export default {
+  props: {
+    listPage: {
+      type: Object,
+      default: () => [],
+    },
+  },
+  methods:{
+    getPageUri(item){
+      return `/${item.siteKey.typeName}/${eXo.env.portal.portalName}/${item.uri}`;
```

Comment:
this will return `/group/spaces/SPACE_GROUP/analytics` for space context, it will not work. you should test on typeName to build an URI using `/${eXo.env.portal.context}/g/...` if group navigation, else `/${eXo.env.portal.context}/{eXo.env.portal.portalName}/${item.uri}`